tests/integration/security/testdata/rbac/v1beta1-jwt.yaml.tmpl

```diff
@@ -55,4 +56,11 @@ spec:
     from:
     - source:
         requestPrincipals: ["*"]
+  - to:
+    - operation:
+        paths: ["/permission"]
+        methods: ["GET"]
+    when:
+    - key: request.auth.claims[permission]
+      values: ["write"]
```

Comment:
Is this common knowledge that we treat `space` as deliminator, and convert string claims into string list?

Reply:
Yeah I think so, see https://github.com/istio/istio/issues/13565 and also https://auth0.com/docs/tokens/guides/access-token/validate-access-token#check-permissions

Reply:
Ok. Let's remember to add this to release note and/or to the post for all authn/authz change.

Reply:
Sure, I'll update this in the 1.4 release notes.